### ISSUE TITLE
[Feature Request] Sort by last updated with unread manga at the top

### ISSUE THREAD

**Issue by ArbitraryFox** [inorichi/tachiyomi#2897]
### Why/User Benefit/User Problem

Currently the app sorts by recently updated. If one has been read, it stays there and the user needs to scroll through the list in case they miss one. 

By having this feature, unread chapters will never be missed.
Yes there is a filter, but it becomes tedious to use since other manga also disappear from the list which makes it look empty.

### What/Requirements

Basically, this is unread + update time mixed into one.
Everything is sorted by last updated.
Unread will be at the top of the list, also sorted by last updated.

There is an app here that filters it that way: https://novelupdates.inverse.cc/

**Comment by arkon** [comment]
Sounds the same as #2740.
